src/pyhf/schemas/1.1.0/defs.json

```diff
@@ -0,0 +1,406 @@
+              "parameters": { "type": "array", "items": {"$ref": "#/definitions/parameter"} }
+          },
+          "additionalProperties": false,
+          "required": ["channels"]
+        },
+        "domain": {
+          "type": "object",
+          "properties": {
+            "name": { "type" : "string" },
+            "type": { "const": "product_domain" },
+            "axes": { "type": "array", "items": {"$ref": "#/definitions/unbinnedAxis"}, "minItems": 1}
+          },
+          "additionalProperties": false,
+          "required": ["name", "type", "axes"]
+        },
+        "init": {
+          "type": "object",
+          "properties": {
+            "name": { "type" : "string" },
+            "data": { "type": "array", "items": {"type": "number"}, "minItems": 1 },
+            "fixed": { "type": "boolean" }
+          },
+          "additionalProperties": false,
+          "required": ["name", "data"]
+        },
+        "dataset": {
+            "type": "object",
+            "properties": {
+                "axes": { "type": "array", "items": {"$ref": "#/definitions/binnedAxis"}, "minItems": 1, "maxItems": 1},
+                "name": { "type": "string" },
+                "type": { "const": "binned" },
+                "contents": { "type": "array", "items": {"type": "number"}, "minItems": 1 }
+            },
+            "required": ["axes", "name", "type", "contents"],
+            "additionalProperties": false
+        },
+        "likelihood": {
+            "type": "object",
+            "properties": {
+                "name": { "type": "string" },
+                "distributions": { "type": "array", "items": { "type": "string" } },
+                "datasets": { "type": "array", "items": { "type": "string" } }
+            },
+            "required": ["name", "distributions", "datasets"],
+            "additionalProperties": false
+        },
+        "measurement": {
+            "type": "object",
+            "properties": {
+                "name": { "type": "string" },
+                "poi": { "type" : "string" },
+                "parameters": { "type": "array", "items": {"$ref": "#/definitions/parameter"} },
+                "domains": { "type": "array", "items": {"type": "string"}},
+                "likelihood": { "type": "string" }
+            },
+            "required": ["name", "poi", "parameters", "likelihood"],
+            "additionalProperties": false
+        },
+        "parameter": {
+            "type": "object",
+            "properties": {
+                "name": { "type": "string" },
+                "auxdata": { "type": "array", "items": {"type": "number"}, "minItems": 1 },
+                "factors": { "type": "array", "items": {"type": "number"}, "minItems": 1 },
+                "sigmas": { "type": "array", "items": {"type": "number"}, "minItems": 1}
+            },
+            "required": ["name"],
+            "additionalProperties": false
+        },
```

Comment:
Is this the object that goes into `parameter_points` in the current HS3 draft? I don't quite follow how to map this to the document. I started looking at this as I as wondering if the format would allow to conveniently distinguish between initial parameter values for the purpose of minimization starting point and default values for the purpose of dataset generation.